### DISCUSSION TITLE
Mathematical constants

### DISCUSSION THREAD

**Discussion by vanillajonathan** [PRQL/prql#3580] [Ideas]
There could be mathematical constants for _pi_, _e_ and _tau_.

Example:
```elm
from numbers
filter number == math.pi
```

https://docs.python.org/3/library/math.html#constants

**Comment by PrettyWood** [comment]
@vanillajonathan pi and e (well `math.exp(1)`) should be available now (see https://github.com/PRQL/prql/pull/3909)
I reckon we can close this discussion
